src/containers/Transactions/simpleTab.scss

```diff
@@ -95,12 +95,12 @@ $subdued-color: $black-40;
       }
     }
 
-    .attestation {
+    .xchainbridge {
```

Comment:
Can you make this more general like SimpleGroup that contains SimpleRows.  This will make it more universal.  See #374.